### ISSUE TITLE
aria-expanded should be true/false rather than 1/0

### ISSUE THREAD

**Issue by sharpchi** [moodleuulm/moodle-local_boostnavigation#111]
I was running the axe accessibility checker on our Moodle page, and it highlighted that aria-expanded 0/1 were invalid values.
e.g. https://github.com/moodle-an-hochschulen/moodle-local_boostnavigation/blob/b493b4975692eab9e2c2383bcce3a0efdcf39cea/amd/src/collapsenavdrawernodes.js#L130

The documentation says true/false. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded

**Comment by abias** [comment]
This plugin has finally reached end of life now and will not be upgraded to Moodle 4.x.

I would like to invite you to have a look at our [Moodle 4 theme theme_boost_union](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union) which supports several aspects of this plugin's feature set and is actively maintained and extended. Thank you for your understanding.

Against this background, I will close this issue.
